src/frontend/Felles/Visittkort/Visittkort.tsx

```diff
@@ -60,6 +75,17 @@ const VisittkortComponent: FC<{
         fullmakt,
         vergemål,
     } = personopplysninger;
+    const skalLenkeTilBehandling =
```

Comment:
```suggestion
    const skalLenkeTilFagsystemBehandling =
```
For å presisere at vi lenker tilbake til fagsystemet